volontulo/lib/email.py

```diff
@@ -20,17 +22,37 @@
 }
 
 
-def send_mail(templates_name, recipient_list, context=None):
+# pylint: disable=unused-argument
+def send_mail(templates_name, recipient_list, context=None, *args, **kwargs):
     u"""Proxy for sending emails."""
+
+    fail_silently = kwargs.get('fail_silently', False)
+    auth_user = kwargs.get('auth_user')
+    auth_password = kwargs.get('auth_password')
+    connection = kwargs.get('connection')
+
     context = Context(context or {})
     text_template = get_template('emails/{}.txt'.format(templates_name))
     html_template = get_template('emails/{}.html'.format(templates_name))
 
-    return django_send_mail(
+    bcc = list(get_administrators_emails().values())
+    connection = connection or get_connection(
+        username=auth_user,
+        password=auth_password,
+        fail_silently=fail_silently
+    )
+    # required, if omitted then no emails from BCC are send
+    headers = {'bcc': ','.join(bcc)}
+    email = EmailMultiAlternatives(
         SUBJECTS[templates_name],
         text_template.render(context),
         FROM_ADDRESS,
         recipient_list,
-        fail_silently=False,
-        html_message=html_template.render(context),
+        bcc,
+        connection=connection,
+        headers=headers
     )
+    if html_template:
```

Comment:
By design all mails will have html template, so there's no need to use if statement.